alacritty/src/config/color.rs

```diff
@@ -29,6 +30,25 @@ impl Colors {
     }
 }
 
+impl Default for Colors {
+    fn default() -> Self {
+        Self {
+            primary: Default::default(),
+            cursor: Default::default(),
+            vi_mode_cursor: Default::default(),
+            selection: Default::default(),
+            normal: Default::default(),
+            bright: Default::default(),
+            dim: Default::default(),
+            indexed_colors: Default::default(),
+            search: Default::default(),
+            line_indicator: Default::default(),
+            hints: Default::default(),
+            opaque_background_colors: true,
```

Comment:
I wonder if it wouldn't be worth it to change this to `transparent_background_colors`, just so the default is `false`.

I also think that people struggle more to understand 'opaque' compared to 'transparent'. Though our existing window opacity does already introduce this concept.